webui/react/src/services/api-ts-sdk/api.ts

```diff
@@ -3152,6 +3152,199 @@ export const V1FittingPolicy = {
+    /**
+     * The time the run ended.
+     * @type {Date | DateString}
+     * @memberof V1FlatRun
+     */
+    endTime?: Date | DateString;
+    /**
+     * The current state of the run(trial).
+     * @type {Trialv1State}
+     * @memberof V1FlatRun
+     */
+    state: Trialv1State;
+    /**
+     * The tags of the run.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    tags: string;
+    /**
+     * The total size of checkpoints.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    checkpointSize: string;
+    /**
+     * The count of checkpoints.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    checkpointCount: number;
+    /**
+     * The type of searcher for the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    searcherType?: string;
+    /**
+     * The searcher metric name for the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    searcherMetric?: string;
+    /**
+     * Signed searcher metrics value.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    searcherMetricValue?: number;
+    /**
+     * The id of external run
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    externalRunId?: number;
+    /**
+     * Trial hyperparameters.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    hyperparameters?: string;
+    /**
+     * summary metrics.
+     * @type {any}
+     * @memberof V1FlatRun
+     */
+    summaryMetrics?: any;
+    /**
+     * The id of the experiment linked to the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    experimentId?: number;
+    /**
+     * The id of the user who created the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    ownerId?: number;
+    /**
+     * Original id of a forked or continued experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    forkedFrom?: number;
+    /**
+     * The id of external experiment
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    externalExperimentId?: number;
+    /**
+     * The resource pool the experiment was created in.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    resourcePool?: string;
+    /**
+     * The current progress of a running experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    experimentProgress?: number;
+    /**
+     * The description of the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    experimentDescription?: string;
+    /**
+     * The experiment name.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    experimentName?: string;
+    /**
+     * The display name of the user that created the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    displayName?: string;
+    /**
+     * The username of the user that created the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    username?: string;
```

Comment:
nit: don't think these are necessary

Reply:
Wait, why do we need to return it in order to sort by it? In the past we've always done sorting on the backend.

Reply:
Hmm. made an incorrect assumption here about Bun() sql queries. I am actually able to remove it. Fixed, soryy